src/inmanta/server/services/resourceservice.py

```diff
@@ -455,7 +455,7 @@ async def resource_action_update(
                 if resource_action is None:
                     # new
                     if started is None:
-                        return 500, {"message": "A resource action can only be created with a start datetime."}
+                        raise Forbidden("A resource action can only be created with a start datetime.")
```

Comment:
500 != forbidden

Reply:
Switched it to 400 here too https://github.com/inmanta/inmanta/pull/2521/commits/cc22de2ff73581609eb4fc3ce55150fba0a6d50b